scenariorun/src/main/kotlin/com/cosmotech/scenariorun/dataset/DatasetUtils.kt

```diff
@@ -32,10 +32,12 @@ internal fun findDatasetsAndConnectors(
     addDatasetAndConnector(
         datasetService, connectorService, organizationId, datasetId, datasets, connectors)
   }
-  val parameterGroupIds = runTemplate.parameterGroups
+  val parameterGroupIds = runTemplate.parameterGroups ?: listOf()
```

Comment:
`runTemplate?.parameterGroups` should be enough here

Reply:
runTemplate cannot be null.